tests/configs/configs_env_validate_test.py

```diff
@@ -0,0 +1,279 @@
+import os
+from typing import Optional
+import pytest
+import requests
+
+from node_cli.configs.env import (
+    absent_params,
+    load_env_file,
+    build_params,
+    populate_params,
+    get_env_config,
+    validate_params,
+    validate_env_type,
+    validate_env_alias_or_address,
+    validate_contract_address,
+    validate_contract_alias,
+    get_chain_id,
+    get_network_metadata,
+    ContractType,
+    ALLOWED_ENV_TYPES,
+)
+from node_cli.utils.exit_codes import CLIExitCodes
+
+
+class FakeResponse:
+    def __init__(self, status_code: int, json_data: Optional[dict] = None):
+        self.status_code = status_code
+        self._json_data = json_data or {}
+
+    def json(self):
+        return self._json_data
+
+
+def test_absent_params_returns_missing_keys():
+    params = {
+        'A': '',
+        'B': 'value',
+        'C': '',
+        'MONITORING_CONTAINERS': 'optional',
+    }
+    missing = absent_params(params)
+    assert 'A' in missing
+    assert 'C' in missing
+    assert 'MONITORING_CONTAINERS' not in missing
+
+
+def test_load_env_file_nonexistent():
+    with pytest.raises(SystemExit) as excinfo:
+        load_env_file('nonexistent.env')
```

Comment:
suggest keeping `load_env_file` like this:

```python
def load_env_file(env_filepath: str) -> None:
    if not load_dotenv(dotenv_path=env_filepath):
        error_exit(f'Failed to load environment from {env_filepath}')
```

and if `load_dotenv` will crash, allow it to do it - exception will be handled on the top level (`node_cli/main.py`)